hbp_knowledge/receptors/matta2017.bel

```diff
@@ -75,308 +75,411 @@ these broadly distributed receptors participate in arousal, motivation,
 4C) indicating that NACHO can enhance intracellular receptor assembly"
+SET Confidence = "High"
 p(HGNC:TMEM35A) -> complex(a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"),a(CHEBI:epibatidine))
+UNSET Confidence
 
 SET Evidence = "Whereas this subunit combination is
 not competent to form a functional channel (Champtiaux et al.,
 2002; Dash et al., 2014; Kuryatov et al., 2000), NACHO still mediated
 partial receptor assembly as reflected by [3H]epibatidine binding (Figure 4D)"
 # this subunit combination: alpha6beta2
+SET Confidence = "High"
 p(HGNC:TMEM35A) -> a(MESH:"alpha6beta2 nicotinic acetylcholine receptor")
 p(HGNC:TMEM35A) -> complex(a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"),a(CHEBI:epibatidine))
+UNSET Confidence
 
 SET Evidence = "This NACHO-mediated [3H]epibatidine binding
 to alpha6beta2 transfectants was also displaced by conotoxin MII (Figure 4E)"
+SET Confidence = "High"
 a(MESH:"alpha-conotoxin MII") -| complex(a(MESH:"alpha6beta2 nicotinic acetylcholine receptor"),a(CHEBI:epibatidine))
+UNSET Confidence
 
 SET Species = "10090"
 
 SET Evidence = "We previously showed that [125I]alpha-bungarotoxin binding is
 absent in NACHO knockout mice brain (Gu et al., 2016),
 which express normal levels of RIC-3 mRNA (Figure S1B)"
+SET Confidence = "High"
 p(MGI:Tmem35a) -> complex(a(MESH:Bungarotoxins),p(FPLX:CHRN))
-
+UNSET Confidence
 # KO of NACHO
 
 SET Evidence = "[125I]epibatidine binding
 was decreased in all brain regions evaluated—it was reduced
 by 66% in cortex, 39% in striatum, and 51% in medial
 vestibular nucleus (MVN) (Figures 5A and 5D)"
 # binding to alpha6-containing receptors
+SET Confidence = "High"
 SET MeSHAnatomy = {"Cerebral Cortex","Corpus Striatum","Vestibular Nuclei"}
 p(MGI:Tmem35a) -> complex(a(CHEBI:epibatidine),a(HBP:HBP00180))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "This partial reduction fits with our data showing that
 NACHO enhances [3H]epibatidine binding to alpha4beta2 and other
 heteromeric nAChRs but that some [3H]epibatidine binding
 occurs to certain heteromeric nAChRs in the absence of NACHO (Figure 2B)"
+SET Confidence = "High"
 p(MGI:Tmem35a) -> complex(a(HBP:HBP00174),a(CHEBI:epibatidine))
+UNSET Confidence
 
 SET Evidence = "Labeling of alpha6-containing nAChRs in striatum by [125I]conotoxin
 MII is virtually abolished in NACHO knockout mice (Figures
 5B and 5D), indicating a vital role for NACHO in assembly of
 these presynaptic receptors"
+SET Confidence = "High"
 SET MeSHAnatomy = "Corpus Striatum"
 p(MGI:Tmem35a) -> a(HBP:HBP00180)
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Consistent with our previous autoradiographic results
 (Gu et al., 2016), we found no detectable [3H]alpha-bungarotoxin
 binding to brain membranes from NACHO knockouts"
+SET Confidence = "Medium"
 SET MeSHAnatomy = "Brain"
-p(MGI:Tmem35a) -> complex(a(MESH:Bungarotoxins),a(GO:membrane))
+p(MGI:Tmem35a) -> complex(a(MESH:Bungarotoxins),loc(GO:membrane))
```

Comment:
Can you use the location term in a complex? I'm not sure about this.
Therefore I coded it as an entity that has part in the complex